components/EditModal.tsx

```diff
@@ -55,7 +56,7 @@ export function EditModal({ gift, closeModal }: EditModal) {
       closeModal={closeModal}
       title="Muokkaa lahjaideaa:"
     >
-      <form onSubmit={(e) => void handleEdit(e)}>
+      <form onSubmit={(e) => errorWrapper(async () => await handleEdit(e))}>
```

Comment:
Oon saattanut tämän kysyä jo useasti, mutta varmistan vielä jotten ymmärtänyt väärin :smile: 

Onko tämmöisessä tilanteessa tarpeellista pitää `await`ia? Tämä on kuitenkin ainoa asia, mitä ajetaan `onSubmit`-funktiossa

Reply:
Kyseisen rivin voinee lyhentää seuraavasti, mutta menee omasta mielestä liian vaikea lukuiseksi verrattuna tuohon mikä nyt on :smile: 
```suggestion
      <form onSubmit={errorWrapper(handleEdit)}>
```

Reply:
Artikkeli oli hyvä, kiitos siitä! :+1: